integration/swarm.bats

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Swarm testing
+
+# Environment variables
+DOCKER_EXE="docker"
+# Image for swarm testing
+nginx_image="gabyct/nginx"
+# Name of service to test swarm
+SERVICE_NAME="testswarm"
+# Maximum number of replicas that will be launch (just a quick test)
+number_of_replicas=4
+# Saves the name of the replicas
+declare -a REPLICAS_UP
+
+setup() {
+	interfaces=($(readlink /sys/class/net/* | grep -i pci | xargs basename -a))
+	swarm_interface_arg=""
+	for i in ${interfaces[@]}; do
+		if [ "`cat /sys/class/net/${i}/operstate`" == "up" ]; then
```

Comment:
is better to use `$()` instead of backquotes for command substitution, this in order to avoid backquotes restrictions

Reply:
@MarioCarrilloA sure